### ISSUE TITLE
New Concept: [Colours]

### ISSUE THREAD

**Issue by jessiepons** [DiGArtefacts/thesaurus#77]
Colours are relevant for Abigail's annotation (e.g. blue lotus, red lotus). Do we want to add a category in our thesaurus? This category can be mapped with the AAT which does cover colours. 

I'm somewhat reluctant to add it as a category because it would would slightly change the scope of our thesaurus (since our thesaurus is one for the description of gandharan buddhist art and... we do not have colours. But perhaps this is an extension that would be valuable for other art history oriented projects (e.g. Kucha) which work on paintings.

What do you think?

@SerenaAutiero @cristiano-moscatelli @frederik-elwert

**Comment by SerenaAutiero** [comment]
I agree that adding possible descriptors such as colours or other classes of adjectives can be an added value looking at possible future uses of the Thesaurus
